locales/de.json

```diff
@@ -2538,19 +2538,44 @@
 			"errorFirstLogin": "Ein Fehler ist bei der Verarbeitung der FirstLogin Daten aufgetreten.",
```

Comment:
Should the emails plain text or a emailto: referer? It is unclear for me by reading the ticket?

_______ 

With adding the emails on this place we have different places where for example mailto:nbc-support@netz-21.de is exists. 
I personally prever one place for one information. Maybe a general
emails: {
"supportNBC": "mailto:nbc-support@netz-21.de"
....
}
That is used on all related places. This result also in a more speakable string. With the cost that we do not see the position in language file. How is your opinion to the topic?

Reply:
I made referer in hbs file: 
![image](https://user-images.githubusercontent.com/26221790/204254746-836617ea-c3cf-444a-aca4-e5769730c4e3.png)

I think that idea related to move emails to separate section is very good. I will let you know when I fix it.